libid/fractals.cpp

```diff
@@ -58,6 +58,7 @@ an appropriate setup, per_image, per_pixel, and orbit routines.
 #include "newton.h"
 #include "parser.h"
 #include "pixel_grid.h"
+#include "fractalb.h"
```

Comment:
remove this when you move the burning ship code to its own file

Reply:
Done